src/numerics/CVodesIntegrator.cpp

```diff
@@ -611,7 +611,7 @@ AnyMap CVodesIntegrator::solverStats() const
                   "supported with sundials versions less than 4.");
     #endif
 
-    #if CT_SUNDIALS_VERION >= 60
+    #if SUNDIALS_VERSION_MAJOR >= 6
```

Comment:
By the way, this line had a typo before, so these lines were not being included, I believe.

Reply:
Gotta love the C preprocessor 🤦